packages/runtime/container-runtime/package.json

```diff
@@ -151,9 +143,11 @@
 		"test": "npm run test:mocha",
 		"test:benchmark:report": "mocha --timeout 999999 --perfMode --parentProcess --fgrep @Benchmark --reporter @fluid-tools/benchmark/dist/MochaReporter.js \"./dist/**/*.perf.spec.*js\"",
```

Comment:
Keep testing CJS intentionally? If so, note in description.

Reply:
'container-runtime' changes were included by mistake.  (removed)